assets/js/components/questionnaires/AddLanguage.jsx

```diff
@@ -33,7 +33,7 @@ class AddLanguage extends Component {
       lang.text.indexOf(value) !== -1 || lang.text.toLowerCase().indexOf(value) !== -1
 
     let languagesOptions = iso6393
-      .filter((lang) => lang.type == "living" && lang.iso6392T != null)
+      .filter((lang) => lang.type == "living")
       .map((lang) => ({ id: idForLanguange(lang), text: lang.name }))
```

Comment:
From [the mapping function](https://github.com/instedd/surveda/blob/9b405132d00d5c849d5081055ebfcd09a0c61ea3/assets/js/components/questionnaires/AddLanguage.jsx#L31), it shows we're not using the `iso6392T` for anything else than trimming down the languages list (exactly what we're reverting with this PR), so the null-check doesn't change a thing in the app's behaviour.